aspnet/hosting/aspnet-core-module.rst

```diff
@@ -0,0 +1,155 @@
+ASP.NET Core Module Configuration Reference
+=============================================
+
+By `Rick Anderson`_ and `Sourabh Shirhatti`_
+
+In ASP.NET Core, the web application is hosted by an external process outside of IIS. The ASP.NET Core Module is an IIS 7.5+ module which is responsible for process management of  ASP.NET Core http listeners and to proxy requests to processes that it manages. This document provides an overview of how to configure the ASP.NET Core Module for shared hosting of ASP.NET Core.
+
+.. contents:: Sections:
+  :local:
+  :depth: 2
+
+Installing the ASP.NET Core Module
+----------------------------------
+
+Install the `.NET Core Windows Server Hosting <http://go.microsoft.com/fwlink/?LinkId=798480>`__ bundle on the server. The bundle will install the .NET Core Runtime, .NET Core Library, and the ASP.NET Core Module.
+
+Configuring the ASP.NET Core Module
+-----------------------------------
+
+The ASP.NET Core Module is configured via a site or application *web.config* file and has its own configuration section within ``system.webServer - aspNetCore``.
+
+Configuration Attributes
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
++---------------------------+----------------------------------------------------+
+| Attribute                 |     Description                                    |
++===========================+====================================================+
+| processPath               | | Required string attribute.                       |
+|                           | |                                                  |
+|                           | | Path to the executable or script that will launch|
+|                           | | a process listening for HTTP requests.           |
+|                           | | Relative paths are supported. If the path        |
+|                           | | begins with '.' the path is considered to be     |
+|                           | | relative to the site root.                       |
+|                           | |                                                  |
+|                           | | There is no default value.                       |
++---------------------------+----------------------------------------------------+
+| arguments                 | | Optional string attribute.                       |
+|                           | |                                                  |
+|                           | | Arguments to the executable or script            |
+|                           | | specified in  **processPath**   .                |
+|                           | |                                                  |
+|                           | | There is no default value.                       |
```

Comment:
Nit: The schema specifies an empty string default value, so I guess that's the same thing.